src/propertiesdialog.cpp

```diff
@@ -106,18 +104,22 @@ PropertiesDialog::PropertiesDialog(QWidget *parent)
             }
         }
     }
-    if (ag.isValid())
+    if (!ag.isEmpty())
     {
         fixedWithSpinBox->setMaximum(ag.width());
         fixedHeightSpinBox->setMaximum(ag.height());
     }
+    fixedWithSpinBox->setMinimum(qMin(300, fixedWithSpinBox->maximum()));
+    fixedHeightSpinBox->setMinimum(qMin(200, fixedHeightSpinBox->maximum()));
```

Comment:
if we could consider that the lowest possible screen resolution we support is about 800x480 or 800x600 the minimum size could be bigger.

Reply:
Right but they're just numbers > 0. I haven't touched them in this patch; just moved them downward in the code and added a qMin() for the sake of logic.

Reply:
Was only a suggestion while we are at

Reply:
And it was a good suggestion. QTerminal can be resized to ridiculously small sizes -- you could almost make it disappear if there was no WM title-bar. I'll open an issue for it.